### ISSUE TITLE
HackHub Logo

### ISSUE THREAD

**Issue by dkua** [HackHub/hackhub.ca#1]
Sooo we might want a cool logo for the group. Anyone have any ideas/design skills?

cc: @wmak @olanmatt

**Comment by olanmatt** [comment]
I'm on it.

**Comment by dkua** [comment]
@olanmatt thought of the idea to use the hacker symbol
![image](https://f.cloud.github.com/assets/1531139/2429149/3a6afbc8-ac6e-11e3-837d-98b461fff777.png)

The St George campus inspiration for this club [http://www.hackeracademy.org/](http://www.hackeracademy.org/) uses the same symbol.

Should we still use it? I mean it's not like one group has to own it.

**Comment by olanmatt** [comment]
I think we should. Lots of people use the same emblem, however it is what we do with it in our implementation that matters. Current working version of logo...

![hackhub](https://f.cloud.github.com/assets/2716841/2429166/2daf5f04-ac6f-11e3-9c19-53cd1b4403f9.png)

**Comment by dkua** [comment]
One requirement is that it needs to look good on a simple t-shirt. Since @wmak really wants shirts for some reason.

We'd also need to pick a font which goes nice with the logo.

**Comment by wmak** [comment]
Yes tshirts would be awesome, between the two logos posted my vote is on the circular one that @olanmatt posted.

**Comment by olanmatt** [comment]
When I was making it, I was thinking about what the sticker would look like. I want a logo that I would be proud to stick on the back of my MacBook. Also the t-shirt idea is awesome. Font... Helvetica Neue?

**Comment by dkua** [comment]
I was thinking a regular san serif font like Helvetica. Or a monospaced font like Ubuntu Mono. On this topic too. What should the main colours be?

**Comment by olanmatt** [comment]
I am good with those fonts, but want to see them in context. UTSC colours?

**Comment by dkua** [comment]
By UTSC colours do you mean the dark red-maroon and white or the UofT blue and white?

**Comment by olanmatt** [comment]
I am refereincing the colours here...

![image](https://f.cloud.github.com/assets/2716841/2429278/16ebb068-ac76-11e3-8985-d5f233b36f90.png)

**Comment by dkua** [comment]
Ah those ones lol. I'm just trying to imagine them as t-shirt/sticker colours and how well they would contrast with the logo.

Also, a no from me on the green and orange. They look puke-y.

**Comment by wmak** [comment]
Why not stick with the UofT blue and white?

**Comment by dkua** [comment]
I do like the UofT navy blue. That light UTSC blue in the logo looks good too. Nice and bright also, stands out.

**Comment by olanmatt** [comment]
![hackhub](https://cloud.githubusercontent.com/assets/2716841/3405069/f377b664-fd78-11e3-9617-dc57a149d590.png)

Went by the UTSC style guide of 2011 (most recent).

**Comment by dkua** [comment]
For some reason the logo still looks off centre vertically.

My major issue with this is there are too many colours. I'd prefer if it was a logo that was two or three colours max. Also makes any form of t-shirt much easier/cheaper to print. And it would look decent in black and white print.

Also the lines of the inner circles are too thin in my opinion. If they were printed on a t-shirt they'd be muted in a sense.

**Comment by olanmatt** [comment]
OK so less colours, fix vertical center, and larger strokes on the inner circles?

**Comment by dkua** [comment]
Pretty much.

**Comment by olanmatt** [comment]
![hackhub](https://cloud.githubusercontent.com/assets/2716841/4158777/de71cd12-3492-11e4-94dd-c355c185ce3e.png)
![hackhub_name](https://cloud.githubusercontent.com/assets/2716841/4158778/de793584-3492-11e4-8dea-fe23831d23f5.png)

From today's edit meeting.

**Comment by dkua** [comment]
Thank you @olanmatt . Logo looks good, I think the HACKHUB text could be thicker?

**Comment by olanmatt** [comment]
Yeah I was thinking I would use two colours for "HACK" and "HUB" to make them distinct. Thoughts?

**Comment by dkua** [comment]
I think keeping them the same would be best. What do you think @wmak

**Comment by wmak** [comment]
Keep them the same, also yeah the text seems a bit thin.

**Comment by olanmatt** [comment]
![hackhub_name](https://cloud.githubusercontent.com/assets/2716841/4179642/48921ce0-36cf-11e4-9c64-6e35eb8a1ef5.png)
How is this?

**Comment by wmak** [comment]
**I like it.**

**Comment by dkua** [comment]
I like it too. Let's go with this. @olanmatt can you put all the materials and stuff in the repo? @wmak can you use those colours and fonts for the site please
